vllm/model_executor/layers/fused_moe/pplx_prepare_finalize.py

```diff
@@ -110,9 +121,11 @@ def prepare(
             out_expert_x_scale=expert_x_scale,
             dp_x=a1q,
             dp_x_scale=a1q_scale,
-            indices=rank_topk_ids,
+            indices=rank_topk_ids.to(torch.uint32),
             bound_m=bound_m,
         )
+        if expert_x_scale is not None:
+            expert_x_scale = expert_x_scale[:, :, 0:1]
```

Comment:
Is this the same as `expert_x_scale.view(-1, -1, 1)`?

Reply:
No, this is taking only one slice from `expert_x_scale`'s last dim. this is related to the scale format required by `dispatch()`.

Reply:
Can you elaborate on that?  I copied the setup of `expert_x_scales` directly from pplx's `test_all_to_all.py` test so I assumed it would have the proper format already.